howto/download_weights.md

```diff
@@ -27,6 +27,59 @@ python scripts/convert_checkpoint.py --model_size 7B
 > **Note**
 > All scripts support argument [customization](customize_paths.md)
 
+### OpenLLaMA
+
+OpenLM Research has released **Apache 2.0 licensed** weights obtained by training LLaMA on the RedPajama dataset.
```

Comment:
For people not familiar with that, maybe change

> RedPajama dataset.

to 

> the 1.2 trillion token open-source [RedPajama dataset](https://github.com/togethercomputer/RedPajama-Data).

Reply:
Good point